Source/WebCore/platform/graphics/cocoa/SourceBufferParserWebM.cpp

```diff
@@ -950,9 +955,11 @@ webm::Status WebMParser::OnFrame(const FrameMetadata& metadata, Reader* reader,
         return Status(Status::kInvalidElementId);
     }
 
+    std::optional<bool> isKey;
```

Comment:
Can't we use a boolean here and default to false?

Reply:
no, no value set means we will look in the bytestream, if a value is set we use the container information.

Reply:
Where do we look in the byte stream. I see `isKey.value_or(false)` below.

Reply:
we don't now. And that limits us to only playing what we generated with the MediaRecorder.